### ISSUE TITLE
Render polygons of open air theatres, amphitheatres, and similar structures as buildings

### ISSUE THREAD

**Issue by euf** [organicmaps/organicmaps#7637]
At the moment `amenity=theatre` and `historic=archaeological_site` is rendered as a POI. 

I suggest enhancing this by rendering as buildings the polygons which have…

 - `theatre:type=open_air`
 - `theatre:type=amphi` 
 - `archaeological_site=amphitheatre` 

An example: https://www.openstreetmap.org/way/141843950 

Another type of addition would be rendering as buildings the following structures: 

 - `amenity=stage` 
 - `leisure=bleachers`
 - `leisure=bandstand`

I suspect these changes wouldn't require adding new categories to the classificator (I might be wrong).

The default height for those objects can be assumed to be 1 meter or 0.5 levels as they are usually just slightly above the ground (see #3832). 

Rendering them is important because they are visually prominent man-made structures and tend to be local landmarks (imagine an open air theatre in a city park or an amphitheater in an archeological park). 

OSM wiki:
 - [theatre:type=open_air](https://wiki.openstreetmap.org/wiki/Tag:theatre:type%3Dopen_air)
 - [theatre:type=amphi](https://wiki.openstreetmap.org/wiki/Tag:theatre:type%3Damphi)
 - [archaeological_site=amphitheatre](https://wiki.openstreetmap.org/wiki/Tag:archaeological_site%3Damphitheatre)
 - [amenity=stage](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dstage)
 - [leisure=bleachers](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dbleachers)
 - [leisure=bandstand](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dbandstand)

**Comment by map-per** [comment]
I don't understand why an open air theatre should be a building.

**Comment by auge8472** [comment]
> I don't understand why an open air theatre should be a building.

Yes, [these](https://commons.wikimedia.org/wiki/File:Brandluckner_Huabn_bei_einer_Auff%C3%BChrung.jpg) [are](https://www.baunetz.de/meldungen/Meldungen-Freilufttheater_in_Antofagasta_im_Norden_Chiles_5257575.html?bild=3) [absolutely](https://commons.wikimedia.org/wiki/File:Arena_Nimes_01.jpg) [not](https://commons.wikimedia.org/wiki/File:Berlin_waldb%C3%BChne.jpg) [buildings](https://commons.wikimedia.org/wiki/File:Plaza_de_Toros_de_Las_Ventas_(Madrid)_03.jpg).

**Comment by map-per** [comment]
If the theatre is a building or there is a building or roof located inside the theatre this should be added to the OSM data with the `building=*` tag.

**Comment by euf** [comment]
I agree, they should not be considered buildings in the classification sense. 

The point is to visually represent them at the map... 
- either as areas with a building color;
- or as low (1 meter) but still prominent building-like physical structures. 

Just like the bridge outlines, they are also man-made objects, which are big enough to be mapped and seen on the map.

**Comment by biodranik** [comment]
1. How are they rendered in different maps?
2. Do we really need polygons? Is rendering only a border and icon/title in the center enough?

**Comment by euf** [comment]
1. I don't know a renderer who does, osm-carto definitely doesn't. 
2. Yes we do need polygons, because the shape of an amphitheatre or a stage is an important visual cue of local topography. When roaming around some archaeological park I find it weird to see empty space in OM while underlying OSM data has perfectly mapped excavation sites, amphitheatres, etc.
